app/helpers/getAppVersion.tsx

```diff
@@ -0,0 +1,17 @@
+import { getVersion, getBuildNumber } from 'react-native-device-info';
+
+import { Platform } from 'react-native';
+
+const getAppVersion = (): string => {
+  const version = getVersion();
+  // Append "ALPHA" to our iOS builds that are 1.0.0, as we use
+  // a separate Alpha TestFlight that is always 1.0.0.
+  // On android we include "ALPHA" directly in the version name.
+  const isAlpha = version === '1.0.0';
+  const appVersion = `${
+    isAlpha && Platform.OS === 'ios' ? 'ALPHA ' : ''
```

Comment:
Mind replacing this with `isPlatformIos` from `Util.js`?

Reply:
Good call!